config/schedule.rb

```diff
@@ -0,0 +1,4 @@
+every 1.day, at: '4:30 am' do
```

Comment:
Our cloud.gov app is on UTC, so this is like 9:30 eastern.

Change it to like 10:30pm and that will be 3:30am eastern.

```suggestion
every 1.day, at: '10:30 pm' do
```

Reply:
makes sense. I wasn't sure it it'd be local or utc